pyproject.toml

```diff
@@ -40,6 +40,7 @@ dependencies = [
     "scikit-learn >= 0.22",
     "python-dateutil >= 2.8.1",
     "scipy >= 1.4.0",
+    "statsmodels >= 0.13.2",
```

Comment:
I disagree with this and think this is a bad idea. We create our open source libraries to have as minimal dependencies as possible. The more core dependencies you rely on, the problematic it can be for someone installing your library, and more likely dependency conflicts are in the future. 

If you need this library for `medcouple`, we should make it an optional library for Woodwork. Users can do `pip install woodwork[stats]` and get this functionality. Additionally, not everyone that uses Woodwork, will want to use `scipy`, `scikit-learn` or `statsmodels`.

Reply:
Also `statsmodels` is a large package.

Reply:
I see, I can make `statsmodels` an optional library for sure. Regarding your second point, should `scipy` and `scikit-learn` be part of an optional installation as well?

Reply:
The medcouple implementation in `statsmodels` is actually fairly basic, we can make our own implementation to avoid importing an entire package just for this. Also I filed this issue [here](https://github.com/alteryx/woodwork/issues/1548) to track the rest

Reply:
nice find!